src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        User newUser = new UserService().findByEmail(email);
+        if (newUser != null && newUser.getPassword().equals(password)) {
+            return true;
+        }
         return false;
```

Comment:
can be simplified a bit :)
```suggestion
        return newUser != null && newUser.getPassword().equals(password);
```